src/bluesky_stomp/messaging.py

```diff
@@ -33,6 +42,12 @@
 
 CORRELATION_ID_HEADER = "correlation-id"
 
+OTLP_EXPORT_ENABLED = environ.get("OTLP_EXPORT_ENABLED", "false").lower() == "true"
```

Comment:
Could: Neater

```suggestion
OTLP_EXPORT_ENABLED = bool(environ.get("OTLP_EXPORT_ENABLED", False))
```

Reply:
This does not work for env var of false as any strimng is True